### PR TITLE
ocaml-cordova.1.0 - via opam-publish

### PR DESCRIPTION
**Pull request by dannywillems**
Binding OCaml to cordova Javascript object.

Binding OCaml to cordova Javascript object. See
https://github.com/dannywillems/ocaml-cordova-plugin-list for the complete list
of bindings to Cordova plugins.

---
* Homepage: https://github.com/dannywillems/ocaml-cordova
* Source repo: https://github.com/dannywillems/ocaml-cordova
* Bug tracker: https://github.com/dannywillems/ocaml-cordova/issues

---

Pull-request generated by opam-publish v0.3.1

### CODE REVIEW

**Comment by dannywillems** [comment]
@avsm Can you check if everything is right so I can run a script for the rest?

**Comment by dannywillems** [comment]
I forgot to make them compatible with the new opam version where compilers version dependencies is in *depends* section. Will *available* section be supported?

File: packages/ocaml-cordova/ocaml-cordova.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,12 @@
+opam-version: "1.2"
+maintainer: "Danny Willems <contact@danny-willems.be>"
+authors: "Danny Willems <contact@danny-willems.be>"
+homepage: "https://github.com/dannywillems/ocaml-cordova"
+bug-reports: "https://github.com/dannywillems/ocaml-cordova/issues"
+license: "LGPL-3.0 with OCaml linking exception"
+dev-repo: "https://github.com/dannywillems/ocaml-cordova"
```
Comment (yallop) [inline packages/ocaml-cordova/ocaml-cordova.1.0/opam]: Could you please add a `.git` suffix to the `dev-repo` field?

```
dev-repo: "https://github.com/dannywillems/ocaml-cordova.git"
```

**Comment by dannywillems** [comment]
@camelus Done.

**Pull request merged**